docs/index.bs

```diff
@@ -3237,18 +3254,19 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                           Note: This only creates a ServiceWorkerGlobalScope because CORS checks require that. It is not expected that implementations will actually create a ServiceWorkerGlobalScope here.
 
                           1. If |response|'s [=response/type=] is "`opaque`", and [=cross-origin resource policy check=] with |globalObject|'s [=environment settings object/origin=], |globalObject|, "", and |response|'s [=filtered response/internal response=] returns <b>blocked</b>, then return null.
+                          1. Set |timingInfo|'s [=service worker timing info/worker final router source=] be set to {{RouterSourceEnum/"cache"}}.
                           1. Return |response|.
                   1. Return null.
               1. Else if |source| is {{RouterSourceEnum/"race-network-and-fetch-handler"}}, and |request|'s [=request/method=] is \`<code>GET</code>\` then:
                   1. If |shouldSoftUpdate| is true, then [=in parallel=] run the [=Soft Update=] algorithm with |registration|.
-                  1. Let |queue| be an empty [=queue=] of [=/response=].
+                  1. Let |queue| be an empty [=queue=] of tuple of [=race result=].
                   1. Let |raceFetchController| be null.
                   1. Let |raceResponse| be a [=race response=] whose [=race response/value=] is "<code>pending</code>".
                   1. Run the following substeps [=in parallel=], but [=abort when=] |fetchController|'s [=fetch controller/state=] is "<code>terminated</code>" or "<code>aborted</code>":
                       1. Set |raceFetchController| to the result of calling [=fetch=] given |request|, with [=fetch/processResponse=] set to the following steps given a [=/response=] |raceNetworkRequestResponse|:
                           1. If |raceNetworkRequestResponse|'s [=response/status=] is [=ok status=], then:
                               1. Set |raceResponse|'s [=race response/value=] to |raceNetworkRequestResponse|.
-                              1. [=queue/Enqueue=] |raceNetworkRequestResponse| to |queue|.
+                              1. [=queue/Enqueue=] |raceNetworkRequestResponse| as [=race result/race response=] and {{RouterSourceEnum/"network"}} as [=race result/used route=] to |queue|.
```

Comment:
I think it is natural to separate this to several steps. e.g.
1. Let |race result| be a new [=race result=] whose [=race result/race response=] is set to |raceNetworkRequestResponse|, and [=race result/used route=] is set to {{RouterSourceEnum/"network"}}.
2. [=queue/Enqueue=] |race result| to |queue|.

Reply:
Right. I have seperated them into two seperate steps.